kratos/includes/io.h

```diff
@@ -349,6 +370,71 @@ class KRATOS_API(KRATOS_CORE) IO
         KRATOS_ERROR << "Calling base class method (WriteConditions). Please check the definition of derived class" << std::endl;
     }
 
+    /**
+     * @brief Reads a new master-slave constraint.
+     * @details This method is intended to be overridden in derived classes. The base class implementation
+     * produces an error message indicating that the derived class should provide its own implementation.
+     * @param rThisNodes The container holding the nodes involved in the constraint.
+     * @param pThisConstraint A pointer to the master-slave constraint object that will be read.
+     * @note Calling this base class method will trigger an error. Ensure that the method is properly defined
+     *       in any derived class.
+     */
+    virtual void ReadNewConstraint(
```

Comment:
Where is it needed? I don't think that it is a good idea to read only one constraint